app/src/main/kotlin/org/koitharu/kotatsu/reader/ui/pager/webtoon/WebtoonRecyclerView.kt

```diff
@@ -98,6 +111,15 @@ class WebtoonRecyclerView @JvmOverloads constructor(
 		listeners.forEach { it.dispatchScroll(this, dy, centerPosition) }
 	}
 
+	fun relayoutChildren() {
+		forEach { child ->
+			(child as WebtoonFrameLayout).target.requestLayout()
+		}
+		detachedViews.forEach {child ->
+			(child as WebtoonFrameLayout).target.requestLayout()
```

Comment:
what reason to request layout of detached view?

Reply:
Without it the detached view's height not changing. My guest is that request layout clear some measure cache and make the view measure again when it attach to the recyclerview again